simpleprophet/README.md

```diff
@@ -1,67 +1,30 @@
 # simpleprophet KPI forecasting models and tools
 
-This directory contains the simpleprophet forecast models and accompanying tools.  The simpleprophet models prioritize simplicity and stability, adding complexity and "bendiness" only when it significantly improves model performance on a holdback period.  More documentation to come.
+This directory contains the simpleprophet forecast models and accompanying tools.  The simpleprophet models prioritize simplicity and stability, adding complexity and "bendiness" only when it significantly improves model performance on a holdback period.
 
-## data.py
+## Usage
 
-Tools for getting metric actual or forecast data from BigQuery.
+The functions for running the forecasting pipeline are in ```pipeline.py```.  The ```update_table``` function can be run daily to add rows to the output table as necessary to incorporate newly available metrics data.  The ```replace_table``` function will clear the output table and regenerate it from scratch.
 
-## modeling.py
+For model-building the code in ```modeling.py``` may be useful.  It includes a function to evaluate a model on a holdout set and provide some useful visualizations.
 
-Tools for model development and evaluation.
+The ```validations.py``` file contains code that produces plots to evaluate model performance and validate the model behavior over time is reasonable.  It can also be used to compare multiple models.
 
-## models.py
+## Modeling Strategy
 
-The current models and training data specification.
+The ```models.py``` file contains the production model specifications.  The models were developed using the fbprophet framework by Jesse McCrosky.  The guiding modeling philosophy was to be guided by simplicity and intuitive fit, while informing the modeling process using conventional types of quantitative evidence.
```

Comment:
"The models were developed using the fbprophet framework by Jesse McCrosky" sounds like fbprophet was created by Jesse. Should be "The models were developed by Jesse McCrosky using the fbprophet framework"

Reply:
Thanks!  Made the fixes.